src/features/steps/SIMCard9151.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+
+import { Back } from '../../common/Back';
+import Link from '../../common/Link';
+import Main from '../../common/Main';
+import { Next } from '../../common/Next';
+
+export default () => (
+    <Main>
+        <Main.Content heading="Plug in iBasis SIM card">
```

Comment:
TIttle still says iBasis. 

"Plug in SIM card"

Reply:
Haha sorry, I didn't pick up on that part before.